actions/imanage/test/helpers.js

```diff
@@ -0,0 +1,132 @@
+const gql = async (query) => {
+  return [{ id: 1, name: 'Bruce Wayne' }];
+};
+
+const fetch = async (url, options) => {
+  if (url === "https://cloudimanage.com/auth/oauth2/token") {
+    return {
+      ok: true,
+      status: 200,
+      json: () => {
+          return {
+            "access_token": "access_token_12345",
+            "refresh_token": "refresh_token",
+            "token_type": "Bearer",
+            "expires_in": 1800
+          }
+        }
+    }
+  }
+
+  if (url === "https://cloudimanage.com/work/api/v2/customers/0000/documents") {
```

Comment:
Split common part of the url with dynamic part and reuse the common part so var = common + dynamic